service/controller/v5/key/key.go

```diff
@@ -91,6 +94,12 @@ func BlobName(customObject providerv1alpha1.AzureConfig, role string) string {
 	return fmt.Sprintf("%s-%s-%s", VersionBundleVersion(customObject), cloudConfigVersion, role)
 }
 
+func CloudConfigSmallTemplates() []string {
```

Comment:
Should be `IgnitionSmallTemplate`? Does it make sense to return a slice here?

Reply:
Same above - we can track that after move ignition to separate repo